src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -274,7 +265,7 @@ private sealed class Mapper : MapperBase
                     var shape = inputNodeInfo.Shape;
                     var inputType = OnnxUtils.OnnxToMlNetType(inputNodeInfo.Type);
 
-                    var inputShape = inputNodeInfo.Shape;
+                    var inputShape = AdjustDimensions(inputNodeInfo.Shape).ToList();
```

Comment:
Would it make sense for AdjustDimensions to return an IEnumerable{int}, and then the caller can turn it into a List or Array as appropriate?
The way it is currently written, this will crest an Array, and then copy to a List.

Reply:
Will change to ienumerable.